.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
Shouldn't this file be on .gitignore?

Reply:
I would keep it out of the gitigore, since these could be useful.